01_prepare_host.sh

```diff
@@ -31,7 +31,7 @@ elif [[ $OS == "centos" || $OS == "rhel" ]]; then
     8)
       sudo dnf config-manager --set-enabled powertools
       sudo dnf install -y epel-release
-      sudo alternatives --set python /usr/bin/python3
+      sudo ln -s /usr/bin/python3 /usr/bin/python || true
```

Comment:
maybe move this and L39 after esac on L45 ?
since the commands are the same for both versions

Reply:
Good point, fixed